core/src/test/scala/partialEvaluator/PartEvalFileSuite.scala

```diff
@@ -0,0 +1,48 @@
+package stainlessfit
+
+import Utils._
+import org.scalatest.funsuite.AnyFunSuite
+
+import core.Core
+import stainlessfit.core.partialEvaluator.PartialEvaluator
+
+class PartEvalFileSuite extends AnyFunSuite {
```

Comment:
Should probably be given the possibility to change debug values

Reply:
Yes please.